maintainers/scripts/copy-tarballs.pl

```diff
@@ -81,17 +99,6 @@
 
     die unless -e $storePath;
 
-    cp($storePath, $dstPath) or die;
+    `tar -cJf $dstPath -C $(dirname $storePath) $storePath` or die;
 
-    my $md5 = hashFile("md5", 0, $storePath) or die;
-    symlink("../$fn", "$tarballsCache/md5/$md5");
-
-    my $sha1 = hashFile("sha1", 0, $storePath) or die;
-    symlink("../$fn", "$tarballsCache/sha1/$sha1");
-
-    my $sha256 = hashFile("sha256", 0, $storePath) or die;
-    symlink("../$fn", "$tarballsCache/sha256/$sha256");
-
-    $sha256 = hashFile("sha256", 1, $storePath) or die;
-    symlink("../$fn", "$tarballsCache/sha256/$sha256");
```

Comment:
Could you maybe explain what the different hashes are generated for? Is it an attempt to store just one file per URL for all possible hashing algorithms? If so, does the storePath not depend on the hashing algorithm so that hashes are re-linked during subsequently processed hashing algorithms of the same ULR? Actually, the hash values only depend on the content and not even on the URL, right? So before I just delete it, what is it for? :)